### DISCUSSION TITLE
Provide way to let EditorPlugin to move the viewport camera

### DISCUSSION THREAD

**Discussion by blackears** [godotengine/godot-proposals#6634] [Editor]
I'm writing an editor plugin and as part of my package, I'd like to be able to move the viewport camera to specific locations to make it easier for the artist to work on the scene.  I can do this temporarily by setting the transform of the camera passed to EditorPlugin._forward_3d_gui_input(), but it jumps back to what it was before as soon as I try to navigate in the viewport.

**Comment by Calinou** [comment]
- This is already being tracked in https://github.com/godotengine/godot-proposals/issues/1302.
